lib/pengine/pe_output.c

```diff
@@ -1159,22 +1160,26 @@ cluster_times_text(pcmk__output_t *out, va_list args) {
     return pcmk_rc_ok;
 }
 
-PCMK__OUTPUT_ARGS("failed-action", "xmlNodePtr")
+PCMK__OUTPUT_ARGS("failed-action", "xmlNodePtr", "unsigned int")
 static int
-failed_action_text(pcmk__output_t *out, va_list args) {
+failed_action_text(pcmk__output_t *out, va_list args)
+{
     xmlNodePtr xml_op = va_arg(args, xmlNodePtr);
+    unsigned int show_opts = va_arg(args, unsigned int);
 
-    char *s = failed_action_string(xml_op);
+    gboolean show_detail = pcmk_is_set(show_opts, pcmk_show_failed_detail);
```

Comment:
This could just be a bool.

Reply:
Makes sense, in fact since html and text share this function now, there's no need for to keep failed_action_string() a separate function, so I'll change that

Reply:
Added commit on top for this

Reply:
You'll want to double check, but I think with the latest patch on top, this one is no longer even necessary.

Reply:
It's preparation for something in progress. That originally was going to be part of this pr but it needs more work